### PR TITLE
Update UG with details for adding internships

### PR DESCRIPTION
**Pull request by lunaroddity**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -71,6 +71,22 @@ Examples:
 * `add c n/Apple p/98765432 e/applehr@example.com d/Top tech company t/tech t/interested`:
   ![add company with all information added](images/addCompany.png)
 
+### Adding an internship to a company contact: `add i`
+
+Adds an internship of interest to the corresponding company.
+
+Format: `add i n/ROLE_NAME d/DESCRIPTION [s/SCHEDULED_INTERVIEW_TIME]`
```
Comment (Nauxe) [inline docs/UserGuide.md]: Is this missing an `INDEX` parameter to determine what company the Internship is under?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -71,6 +71,22 @@ Examples:
...
+* Internship's `ROLE_NAME` and `DESCRIPTION` are compulsory fields.
+* `SCHEDULED_INTERVIEW_TIME` is optional and can only be used once.
+
+Restrictions:
+* `ROLE_NAME` and `DESCRIPTION` must be alphanumeric.
```
Comment (Nauxe) [inline docs/UserGuide.md]: I think no need alphanumeric? Otherwise your role name and description can't have spaces in it.

**Review by Nauxe — COMMENTED** [review]
Some nits here and there

File: docs/UserGuide.md (hunk 2 above)
Comment (oeggy03) [reply docs/UserGuide.md]: Both can be any non-empty string and should not be blank

for internships

File: docs/UserGuide.md (hunk 1 above)
Comment (lunaroddity) [reply docs/UserGuide.md]: Thanks for catching this

**Comment by Tim-Siu** [comment]
A timely update to the UG! All the concerns have been resolved.

**Review by Tim-Siu — APPROVED** [review]
A timely update to the UG!

**Pull request merged**